nano/core_test/node.cpp

```diff
@@ -1851,21 +1851,23 @@ TEST (node, rep_remove)
 	}
 
 	// Create inactive TCP channel for Rep1
-	std::shared_ptr<nano::transport::channel> channel_rep1 (std::make_shared<nano::transport::channel_tcp> (searching_node, std::weak_ptr<nano::socket> ()));
+	auto channel_rep1 (std::make_shared<nano::transport::fake::channel> (searching_node));
+	channel_rep1->set_endpoint (nano::endpoint (boost::asio::ip::address_v6::loopback (), nano::test::get_available_port ()));
+	channel_rep1->set_node_id (keys_rep1.pub);
```

Comment:
this line can be rmeoved

Reply:
I wanted to make it explicit that this channel belongs to rep1. By removing these lines the channel would have searching_node's id and endpoint and that would be confusing I thought. But I'm fine with removing them.

Reply:
If you think they are important then OK, leave them.

Reply:
Maybe it's not that important. I've removed them.

Reply:
Oh, the line setting the endpoint was required for fixing the race  condition. I'll add it again when I'm back at my PC